README.dev.md

```diff
@@ -111,54 +111,61 @@ Otherwise, please ensure check both linting (`ruff fix .`) and formatting (`ruff
-
-To [test snippets](https://www.sphinx-doc.org/en/master/usage/extensions/doctest.html) in documentation run
-
-```shell
-cd docs
-make doctest
-```
-
 ## Making a release
 
+### Automated release workflow:
+
+0. **IMP0RTANT:** Create a PR for the release branch and make sure that all checks pass!
+   - if everything goes well, this PR will automatically be closed.
+1. Navigate to [Draft Github Release](https://github.com/EIT-ALIVE/eitprocessing/actions/workflows/release_github.yml)
+   on the [Actions](https://github.com/EIT-ALIVE/eitprocessing/actions) tab.
+2. On the right hand side, you can select the level increase (patch, minor, or major) and which branch to release from.
+   - if released from a different branch than `develop`, then the workflow will attempt to merge the changes into develop
+     as well. If succesfull, the release branch will be deleted from the remote repository.
+   - [follow semantic versioning conventions](https://semver.org/) to chose the level increase:
+     - `patch`: when backward compatible bug fixes were made
+     - `minor`: when functionality was added in a backward compatible manner
+     - `major`: when API-incompatible changes have been made
+3. Visit [Actions](https://github.com/EIT-ALIVE/eitprocessing/actions) tab to check whether everything went as expected.
+4. Navigate to the [Releases](https://github.com/EIT-ALIVE/eitprocessing/releases) tab and click on the newest draft
+   release (which was just automatically generated)
+5. Click on the edit (pencil) icon on the right side of the draft release.
+6. Check/adapt the release notes and make sure that everything is as expected.
+7. Check that "Set as the latest release is checked".
+8. Click green "Publish Release" button to convert the draft to a published release on GitHub.
+   - This will automatically trigger [another GitHub workflow](https://github.com/EIT-ALIVE/eitprocessing/actions/workflows/release.yml) that will take care of publishing the package on PyPi.
+
+#### Updating the token:
+
+NOTE: the current token (associated to @DaniBodor) allowing to bypass branch protection will expire on June 20th, 2025. To update the token do the following:
+
+1. [Create a personal access token](https://github.com/settings/tokens/new) from a GitHub user account with admin
+   priviliges for this repo.
+2. Check all the "repo" boxes and the "workflow" box, set an expiration date, and give the token a note.
+3. Click green "Generate token" button on the bottom
+4. Copy the token immediately, as it will not be visible again later.
+5. Navigate to the [secrets settings](https://github.com/EIT-ALIVE/eitprocessing/settings/secrets/actions).
+6. Edit the `GH_PAT` key giving your access token as the new value.
+
+### Manually create a release:
+
 0. Make sure you have all required developers tools installed `pip install -e .'[dev]'`
 1. Branch from `main` and prepare the branch for the release (e.g., removing the unnecessary files, fix minor bugs if necessary).
```

Comment:
This happens in step 4 below, or do I misunderstand your meaning? I clarified a bit in 2ea4890, let me know if this is clear enough

Reply:
Step 4 merges the "release" branch. The "release" branch generally is `develop`. If you want to manually release a feature (e.g. branch `fun_feature`), you first have to merge `fun_feature` into `develop`, and then start at 0.

Step 0 suggests that there should be a separate branch `release` branched from `main`. Does that mean that a feature should be merged into `release` instead of `develop`?

Reply:
Isn't this covered by the branching strategy mentioned above (that you wrote)?